system/lib/libc/musl/src/thread/__timedwait.c

```diff
@@ -95,11 +94,13 @@ int __timedwait_cp(volatile int *addr, int val,
 	r = -__futex4_cp(addr, FUTEX_WAIT|priv, val, top);
 #endif
 	if (r != EINTR && r != ETIMEDOUT && r != ECANCELED) r = 0;
+#ifndef __EMSCRIPTEN__ // XXX Emscripten revert musl commit a63c0104e496f7ba78b64be3cd299b41e8cd427f
```

Comment:
What is this change doing?   Can/should it split into its own PR?

Reply:
It was excluding code that is not related to Emscripten. Reverted with commit 6ea3ad394b1b870f157be7162b7546f64a8896e3 for now.